CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 3.0.0 (2019-03-27)
+
+## Breaking changes
+
+Remove glyphicons in favor of material icons [21518f6](https://github.com/AmpleOrganics/Blaze.vue/commit/21518f6a2602b3b3a73eda09c68a70c4595d6501)
+
+## New
+
+Alert: add a close button to ao-alert to allow early dismissal [6555e6e](https://github.com/AmpleOrganics/Blaze.vue/commit/6555e6ee916e1ad825c5312dad94058b866310a8)
```

Comment:
Items under `New` should have a line break for readability
![image](https://user-images.githubusercontent.com/9040052/55098400-10dfe000-5094-11e9-82a8-2c82ca7b3f86.png)

Reply:
fixed for `new` and `fixes` items